test/integration/an_existing_user_can_check_out_to_place_an_order_test.rb

```diff
@@ -0,0 +1,44 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.